mm2src/mm2_metrics/src/mm_metrics.rs

```diff
@@ -442,8 +442,8 @@ mod test {
                     "count": 2.0,
                     "key": "rpc.query.spent_time",
                     "labels": { "coin": "KMD", "method": "blockchain.transaction.get" },
-                    "max": 1.0,
-                    "min": 1.0,
+                    "max": 2.0,
+                    "min": 2.0,
```

Comment:
That's really strange that we passed two timing metrics: `1s` once and `2s` once. So we expect to get `max: 2.0` and `min: 1.0`.
Please try to debug this to figure out how to fix the bug.

Reply:
yes fixed...my bad as I was using one delta for the timing metrics. corrected. Thanks